lib/Runtime/Types/DictionaryTypeHandler.cpp

```diff
@@ -869,6 +869,81 @@ namespace Js
     }
 
     template <typename T>
+    BOOL DictionaryTypeHandlerBase<T>::DeleteProperty(DynamicObject *instance, JavascriptString *propertyNameString, PropertyOperationFlags propertyOperationFlags)
```

Comment:
Yes, let me look into refactoring that.

---
In reply to: [78818835](https://github.com/Microsoft/ChakraCore/pull/1542#discussion_r78818835) [](ancestors = 78818835,78818261,78817010,78791124)